Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```
@@ -2618,27 +2146,17 @@ script:
       description: The incident score.
       type: Number
   - arguments:
-    - default: false
-      description: The endpoint ID.
-      isArray: false
+    - description: The endpoint ID.
       name: id
-      required: false
-      secret: false
     - default: true
       description: The endpoint IP address.
-      isArray: false
       name: ip
-      required: false
-      secret: false
-    - default: false
-      description: The endpoint hostname.
-      isArray: false
+    - description: The endpoint hostname.
       name: hostname
-      required: false
+    - default: false
+      required: true
```

```suggestion
      default: false
      required: false
```
Something is weird here with the changes.. I think this should be under 'hostname' (also secret)

---

I have made these changes